Updated documents, fixed English Translation (#1400)
Ralim/IronOS: Documentation/Settings.md

> ### Setting: Power source

Reviewer: Please follow the warning at the top of this file and _dont edit it directly_.
Instead edit the english translations and re-run the generation tooling to update this file.

When the device is powered by a battery, this adjusts the low voltage threshold for when the unit should turn off the heater to protect the battery from over-draining.

On device help text:

> Set cutoff voltage to protect battery over-drain. (DC 10V) (S 3.3V per cell, disable power limit)

Reviewer 2: ```suggestion
Set cutoff voltage to protect battery against over-drain. (DC 10V) (S 3.3V per cell, disable power limit)
```

### Setting: Sleep timeout

> The time required with no movement or button-pressing before the device drops down to the sleep temperature.

Reviewer 2: ```suggestion
The time it takes before entering sleep mode while there is no motion or key press.
```

On device help text:

Interval before "sleep mode" starts (S=seconds | M=minutes)

### Setting: Shutdown timeout

> The time required with no movement or button-pressing before the device shuts off the tip heater completely and returns to the main idle screen.

Reviewer 2: ```suggestion
The amount of time, without movement or button presses, before the device shuts off the tip heater completely and returns to the main idle screen.
```

Scale of how sensitive the device is to movement. Higher numbers == more sensitive. 0 == motion detection turned off.

Adjust the90 device help text:

0=off | 1=least sensitive | ... | 9=most sensitive

### Setting: Temperature unit

> Select a preference of °C or °F.

Reviewer 2: ```suggestion
Select the unit in which temperatures should be displayed, °C or °F.
```

On device help text:

> O=off | S=heat up to soldering temp | Z=standby at sleep temp until moved | R=standby at no heat heat until moved

Reviewer 2: Please leave this as is, since in the past I was asked to reduce the length.
```suggestion
O=off | S=soldering temp | Z=standby at sleep temp until moved | R=standby without heating until moved
```

### Setting: Cooldown flashing

If the idle screen should blink the tip temperature for attention while the tip is over 50°C. Intended as a 'tip is still hot' warning.

On device help text:

> Flash the temperature reading, if tip is still hot, after halting heat

Reviewer 2: ```suggestion
Flash the temperature reading after heating was halted while the tip is still hot
```

What is wrong about the current sentence?
It was a really tedious job to make all the 'scrolling texts' as short as possible while preserving the clarity of the content.

### Setting: Calibrate tip CJC?

Used to calibrate the ADC + Op-amp offsets for the tip. This calibration should only be performed when the tip and device temperature are **equal** and at **room/ambient temperature**. 

Generally *not required unless your device is reading more than 5 °C off target temperature*. This is intended to be performed at boot-up before internal components get warm. Avoid repeating the calibration, but if you must repeat it, unplug the unit, and wait for everything to go back to room temperature. Temperatures start to drift even after plugging in the device, which is why cold junction calibration is best done at a fresh boot-up.

On device help text:
- Tip Cold Junction Compensation; not needed if temp is +/- 5 C
- Affirm the tip & device are at room temperature before starting!

On device help text:

Start tip temperature offset calibration

### Setting: Restore default settings?

> Resets settings and calibrations to defaults included with the installed firmware version. Does NOT erase custom user boot up logo's.

Reviewer 2: ```suggestion
Resets settings and calibrations to defaults included in the installed firmware version. Does NOT erase custom user boot up logo's.
```

Flash temperature reading at idle if tip is hot

### Setting: Calibrate tip CJC

Used to calibrate the ADC+Op-amp offsets for the tip. This calibration must be performed when the tip temperature and the handle temperature are equal. Generally not required unless your device is reading more than 5°C off target.

On device help text:

> Tip-temperature Cold Junction Compensation (not needed if Tip is +/- 5 C of display)

Reviewer: Fixed in [307b6c7](https://github.com/Ralim/IronOS/commit/307b6c76f8d6cb42ad59d5e91729fdce0caee773)

### Setting: Swap + - keys

> Swaps which button for increments / decrements on the temperature change screens.

Reviewer 2: ```suggestion
Swaps the button to increment with the button to decrement the temperature in the temp adjust menu.
```

On device help text:

Power intensity of keep-awake-pulse (watt)

### Setting: Hall sensor sensitivity

If the unit has a hall effect sensor (Pinecil), this adjusts how sensitive it is at detecting a magnet to put the device into sleep mode.

On device help text:

> Sensitivity to magnets (0=off | 1=least sensitive | ... | 9=most sensitive)

Reviewer 2: ```suggestion
Sensitivity of the Hall effect sensor to trigger sleep mode (0=off | 1=least sensitive | ... | 9=most sensitive)
```

On device help text:

> Minimum allowed volts per cell (3S: 3 - 3.7V | 4S-6S: 2.4 - 3.7V)

Reviewer 2: Please leave it as it is.
There is nothing wrong about the sentence here!
If you want to add the 'S' after 4, you should also add the 'V' after 3 and after 2.4.
But if you were aware that this was the original state, then you would know that this change was made on purpose to save space/decrease the time to read the 'scrolling text'.

Reviewer: "volts" is slang, it would be perferable to keep "voltage" if we can

### Setting: Anim. speed

> How fast should the menu animations loop or off setting.

Reviewer 2: Please leave it as it is.
Since the sentence you propose doesn't make any sense.

On device help text:

Adjust the OLED screen brightness

### Setting: Invert screen

Inverts the entire OLED.

On device help text:

Invert the OLED screen colors

### Setting: Boot logo duration

> Sets the duration for the boot logo (S=seconds).

Reviewer 3: S should be lowercase

Reviewer: Fixed in [307b6c7](https://github.com/Ralim/IronOS/commit/307b6c76f8d6cb42ad59d5e91729fdce0caee773)

On device help text:

Adjust the OLED screen brightness

### Setting: Invert screen

Inverts the entire OLED.

On device help text:

> Invert OLED screen colours Black | White

Reviewer 2: ```suggestion
Invert OLED screen colours
```

There are only 2 colors!
How does adding 'Black | White' help us?
